app/services/fastly_vcl/whitelisted_params.rb

```diff
@@ -0,0 +1,59 @@
+module FastlyVCL
+  # Handles updates to our VCL snippet on Fastly that whitelists params
+  class WhitelistedParams
+    VCL_DELIMITER_START = "^(".freeze
+    VCL_DELIMITER_END = ")$".freeze
+    SNIPPET_NAME = "Whitelist certain querystring parameters".freeze
+    FILE_PARAMS = YAML.load_file("config/fastly/whitelisted_params.yml").sort.freeze
```

Comment:
This done for 2 reasons:
1. Keep things DRY.
2. Make it adaptable for future use. Taking inspiration from Molly's work on Elasticsearch, if we start to leverage Fastly more and add more snippets, these constants can be refactored into a base class.